oci/private/image.sh

```diff
@@ -33,11 +33,17 @@ function base_from() {
   local path="$1"
   # shellcheck disable=SC2045
   for blob in $(ls -1 -d "$path/blobs/"*/*); do
-    local relative=${blob#"$path/"}
-    mkdir -p "$OUTPUT/$(dirname "$relative")"
-    cat "$blob" >"$OUTPUT/$relative"
+    local relative_to_blobs=${blob#"$path/blobs"}
+    mkdir -p "$OUTPUT/blobs/$(dirname "$relative_to_blobs")"
+    if [[ "$USE_TREEARTIFACT_SYMLINKS" == "1" ]]; then
+      # Relative path from `output/blobs/sha256/` to `$blob`
+      relative=$(coreutils realpath --relative-to="$OUTPUT/blobs/sha256" "$blob" --no-symlinks)
+      coreutils ln -s "$relative" "$OUTPUT/blobs/$relative_to_blobs"
+    else
+      coreutils cat "$blob" > "$OUTPUT/blobs/$relative_to_blobs"
+    fi
   done
-  cat "$path/oci-layout" >"$OUTPUT/oci-layout"
+  coreutils cat "$path/oci-layout" >"$OUTPUT/oci-layout"
   jq '.manifests[0].annotations["org.opencontainers.image.ref.name"] = "intermediate"' "$path/index.json" >"$OUTPUT/index.json"
```

Comment:
jq from the PATH here? not from the `jq.jqinfo.bin,` toolchain?

Reply:
hermetic jq is already in the PATH, added at the top of the script.